Source/DafnyDriver/VerificationResultConvertor.cs

```diff
@@ -0,0 +1,126 @@
+//-----------------------------------------------------------------------------
+//
+// Copyright (C) Microsoft Corporation.  All Rights Reserved.
+// Copyright by the contributors to the Dafny Project
+// SPDX-License-Identifier: MIT
+//
+//-----------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Boogie;
+using Microsoft.VisualStudio.TestPlatform.Extensions.TrxLogger;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+using VC;
+
+namespace Microsoft.Dafny {
+
+  /// <summary>
+  /// Utility to translate verification results into test logger events,
+  /// allowing us to deliver them through common loggers on the .NET
+  /// platform. For now we support three formats:
+  ///  * TRX files, which can be understood and visualized by various .NET tools.
+  ///  * CSV files, which are easier to parse and summarize. 
+  ///  * human-readable text output.
+  /// </summary>
+  public static class VerificationResultConvertor {
```

Comment:
From the name I can see that it does something with VerificationResults. I think in this case the conversion target (TestLoggerEvents) is more specific than the source (VerificationResult), so I'd prefer to base the name on that, or put both in the name.

Reply:
Yeah, that's a good point. One thing that makes me hesitate, though, is that one of the loggers doesn't go through the `ITestLogger` interface. Although I'm thinking that maybe `VerificationResultLogger` is a better name in either case.